services/QuillLMS/app/models/change_log.rb

```diff
@@ -80,16 +80,16 @@ class ChangeLog < ApplicationRecord
     'Standard',
     'StandardLevel',
     'StandardCategory',
-    'Comprehension::Activity',
-    'Comprehension::Prompt',
-    'Comprehension::Rule',
-    'Comprehension::Passage',
-    'Comprehension::AutomlModel',
-    'Comprehension::Label',
-    'Comprehension::Feedback',
-    'Comprehension::Highlight',
-    'Comprehension::RegexRule',
-    'Comprehension::PlagiarismText'
+    'Evidence::Activity',
```

Comment:
This will require a data migration, correct? Will that be part of this PR?

Reply:
Good catch on these.  I've added a `rake` task to handle namespaces in both `ChangeLog` and `FeedbackHistory` polymorphic references.